recipes/batchgenerators/python3.patch

```diff
@@ -0,0 +1,21 @@
+From d02c1f641b672f23cdddda014bc66ed22874ef13 Mon Sep 17 00:00:00 2001
+From: Jan Janssen <janssen@mpie.de>
+Date: Thu, 28 Nov 2024 11:44:59 +0100
+Subject: [PATCH] remove unittest2
+
+---
+ setup.py | 1 -
+ 1 file changed, 1 deletion(-)
+
+diff --git a/setup.py b/setup.py
+index 6c0a50e..6fbb14c 100755
+--- a/setup.py
++++ b/setup.py
+@@ -17,7 +17,6 @@
+             "scikit-learn",
+             "future",
+             "pandas",
+-            "unittest2",
```

Comment:
These changes are currently discussed in a pull request up-stream https://github.com/MIC-DKFZ/batchgenerators/pull/130 beyond other things.

Reply:
Can you add that link and comment in the recipe? It will be easier to find them there than in this PR when we need it.

Reply:
@ocefpaf Sure - done